
2019-08-09 09:56:59,813 INFO: errors [in /home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py:502]
2019-08-09 13:42:59,313 INFO: errors [in /home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py:501]
2019-08-09 13:44:04,353 ERROR: Exception on /artists/create [GET] [in /home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py", line 397, in create_artist_form
    form = ArtistForm()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 212, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 158, in __init__
    super(Form, self).__init__(*args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 88, in __init__
    super(FlaskForm, self).__init__(formdata=formdata, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 278, in __init__
    self.process(formdata, obj, data=data, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 132, in process
    field.process(formdata)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/csrf/core.py", line 43, in process
    self.current_token = self.csrf_impl.generate_csrf_token(self)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 134, in generate_csrf_token
    token_key=self.meta.csrf_field_name
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 35, in generate_csrf
    message='A secret key is required to use CSRF.'
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 121, in _get_config
    raise KeyError(message)
KeyError: 'A secret key is required to use CSRF.'
2019-08-09 13:44:10,240 ERROR: Exception on /artists/create [GET] [in /home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py", line 397, in create_artist_form
    form = ArtistForm()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 212, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 158, in __init__
    super(Form, self).__init__(*args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 88, in __init__
    super(FlaskForm, self).__init__(formdata=formdata, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 278, in __init__
    self.process(formdata, obj, data=data, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 132, in process
    field.process(formdata)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/csrf/core.py", line 43, in process
    self.current_token = self.csrf_impl.generate_csrf_token(self)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 134, in generate_csrf_token
    token_key=self.meta.csrf_field_name
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 35, in generate_csrf
    message='A secret key is required to use CSRF.'
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 121, in _get_config
    raise KeyError(message)
KeyError: 'A secret key is required to use CSRF.'
2019-08-09 13:44:13,754 ERROR: Exception on /venues/create [GET] [in /home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py", line 191, in create_venue_form
    form = VenueForm()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 212, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 158, in __init__
    super(Form, self).__init__(*args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 88, in __init__
    super(FlaskForm, self).__init__(formdata=formdata, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 278, in __init__
    self.process(formdata, obj, data=data, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 132, in process
    field.process(formdata)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/csrf/core.py", line 43, in process
    self.current_token = self.csrf_impl.generate_csrf_token(self)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 134, in generate_csrf_token
    token_key=self.meta.csrf_field_name
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 35, in generate_csrf
    message='A secret key is required to use CSRF.'
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 121, in _get_config
    raise KeyError(message)
KeyError: 'A secret key is required to use CSRF.'
2019-08-09 13:44:15,544 ERROR: Exception on /shows/create [GET] [in /home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py", line 466, in create_shows
    form = ShowForm()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 212, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 158, in __init__
    super(Form, self).__init__(*args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 88, in __init__
    super(FlaskForm, self).__init__(formdata=formdata, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 278, in __init__
    self.process(formdata, obj, data=data, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 132, in process
    field.process(formdata)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/csrf/core.py", line 43, in process
    self.current_token = self.csrf_impl.generate_csrf_token(self)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 134, in generate_csrf_token
    token_key=self.meta.csrf_field_name
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 35, in generate_csrf
    message='A secret key is required to use CSRF.'
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 121, in _get_config
    raise KeyError(message)
KeyError: 'A secret key is required to use CSRF.'
2019-08-09 13:44:23,934 ERROR: Exception on /artists/create [GET] [in /home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py", line 397, in create_artist_form
    form = ArtistForm()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 212, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 158, in __init__
    super(Form, self).__init__(*args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 88, in __init__
    super(FlaskForm, self).__init__(formdata=formdata, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 278, in __init__
    self.process(formdata, obj, data=data, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 132, in process
    field.process(formdata)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/csrf/core.py", line 43, in process
    self.current_token = self.csrf_impl.generate_csrf_token(self)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 134, in generate_csrf_token
    token_key=self.meta.csrf_field_name
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 35, in generate_csrf
    message='A secret key is required to use CSRF.'
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 121, in _get_config
    raise KeyError(message)
KeyError: 'A secret key is required to use CSRF.'
2019-08-09 13:45:10,306 ERROR: Exception on /artists/create [GET] [in /home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py", line 397, in create_artist_form
    form = ArtistForm()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 212, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 158, in __init__
    super(Form, self).__init__(*args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 88, in __init__
    super(FlaskForm, self).__init__(formdata=formdata, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 278, in __init__
    self.process(formdata, obj, data=data, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 132, in process
    field.process(formdata)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/csrf/core.py", line 43, in process
    self.current_token = self.csrf_impl.generate_csrf_token(self)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 134, in generate_csrf_token
    token_key=self.meta.csrf_field_name
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 35, in generate_csrf
    message='A secret key is required to use CSRF.'
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 121, in _get_config
    raise KeyError(message)
KeyError: 'A secret key is required to use CSRF.'
2019-08-09 13:46:00,547 INFO: errors [in /home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py:501]
2019-08-09 13:46:02,955 ERROR: Exception on /artists/create [GET] [in /home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py", line 397, in create_artist_form
    form = ArtistForm()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 212, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 158, in __init__
    super(Form, self).__init__(*args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 88, in __init__
    super(FlaskForm, self).__init__(formdata=formdata, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 278, in __init__
    self.process(formdata, obj, data=data, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 132, in process
    field.process(formdata)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/csrf/core.py", line 43, in process
    self.current_token = self.csrf_impl.generate_csrf_token(self)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 134, in generate_csrf_token
    token_key=self.meta.csrf_field_name
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 35, in generate_csrf
    message='A secret key is required to use CSRF.'
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 121, in _get_config
    raise KeyError(message)
KeyError: 'A secret key is required to use CSRF.'
2019-08-09 13:49:30,402 INFO: errors [in /home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py:501]
2019-08-09 13:49:34,191 ERROR: Exception on /artists/create [GET] [in /home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py", line 397, in create_artist_form
    form = ArtistForm()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 212, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 158, in __init__
    super(Form, self).__init__(*args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 88, in __init__
    super(FlaskForm, self).__init__(formdata=formdata, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 278, in __init__
    self.process(formdata, obj, data=data, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 132, in process
    field.process(formdata)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/csrf/core.py", line 43, in process
    self.current_token = self.csrf_impl.generate_csrf_token(self)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 134, in generate_csrf_token
    token_key=self.meta.csrf_field_name
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 35, in generate_csrf
    message='A secret key is required to use CSRF.'
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 121, in _get_config
    raise KeyError(message)
KeyError: 'A secret key is required to use CSRF.'
2019-08-09 13:49:54,472 INFO: errors [in /home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py:501]
2019-08-09 13:49:55,818 ERROR: Exception on /artists/create [GET] [in /home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py", line 397, in create_artist_form
    form = ArtistForm()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 212, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 158, in __init__
    super(Form, self).__init__(*args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 88, in __init__
    super(FlaskForm, self).__init__(formdata=formdata, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 278, in __init__
    self.process(formdata, obj, data=data, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 132, in process
    field.process(formdata)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/csrf/core.py", line 43, in process
    self.current_token = self.csrf_impl.generate_csrf_token(self)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 134, in generate_csrf_token
    token_key=self.meta.csrf_field_name
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 35, in generate_csrf
    message='A secret key is required to use CSRF.'
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 121, in _get_config
    raise KeyError(message)
KeyError: 'A secret key is required to use CSRF.'
2019-08-09 13:52:33,348 INFO: errors [in /home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py:501]
2019-08-09 13:52:36,386 ERROR: Exception on /artists/create [GET] [in /home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py", line 397, in create_artist_form
    form = ArtistForm()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 212, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 158, in __init__
    super(Form, self).__init__(*args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 88, in __init__
    super(FlaskForm, self).__init__(formdata=formdata, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 278, in __init__
    self.process(formdata, obj, data=data, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 132, in process
    field.process(formdata)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/csrf/core.py", line 43, in process
    self.current_token = self.csrf_impl.generate_csrf_token(self)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 134, in generate_csrf_token
    token_key=self.meta.csrf_field_name
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 35, in generate_csrf
    message='A secret key is required to use CSRF.'
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 121, in _get_config
    raise KeyError(message)
KeyError: 'A secret key is required to use CSRF.'
2019-08-09 13:54:59,006 INFO: errors [in /home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py:501]
2019-08-09 13:55:52,426 INFO: errors [in /home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py:503]
2019-08-09 13:55:56,987 INFO: errors [in /home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py:503]
2019-08-09 13:56:29,574 INFO: errors [in /home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py:503]
2019-08-09 13:56:39,132 INFO: errors [in /home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py:503]
2019-08-09 13:56:43,179 INFO: errors [in /home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py:503]
2019-08-09 13:56:47,232 ERROR: Exception on /artists/create [GET] [in /home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py", line 399, in create_artist_form
    form = ArtistForm()
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 212, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 158, in __init__
    super(Form, self).__init__(*args, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/form.py", line 88, in __init__
    super(FlaskForm, self).__init__(formdata=formdata, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 278, in __init__
    self.process(formdata, obj, data=data, **kwargs)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/form.py", line 132, in process
    field.process(formdata)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/wtforms/csrf/core.py", line 43, in process
    self.current_token = self.csrf_impl.generate_csrf_token(self)
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 134, in generate_csrf_token
    token_key=self.meta.csrf_field_name
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 35, in generate_csrf
    message='A secret key is required to use CSRF.'
  File "/home/05674711445/Desenvolvimento/Python/venv/fyyur/lib/python3.6/site-packages/flask_wtf/csrf.py", line 121, in _get_config
    raise KeyError(message)
KeyError: 'A secret key is required to use CSRF.'
2019-08-09 13:59:36,056 INFO: errors [in /home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py:504]
2019-08-09 14:01:09,918 INFO: errors [in /home/05674711445/Desenvolvimento/pessoal/udacity/fyyur-fullstacknd/app.py:504]
